examples/connector/python/connector/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+python3 -m venv connector_run
+source connector_run/bin/activate
+mkdir -p protos
+cp ../../../../*.proto protos/
+pip install -r requirements.txt
+python -m grpc_tools.protoc \
```

Comment:
is it possible to have the generated `.py` and `.pyi` files inside a directory? that would make the package cleaner